lib/commands/tickets.js

```diff
@@ -14,6 +14,10 @@ class Tickets {
      * @param query channelId string to send message to
      */
     async execute(query) {
+        if (!query) {
```

Comment:
Are we checking in `.js` files? Probably a good chance to start removing those from version control...

Reply:
this was my bad, should be fixed now.